project_one/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" type="text/css" href="mystyle.css">
+<head>
+	<title></title>
+</head>
+<body>
+    <p id = "pid"> This is section one 
+        <button id = "grading-element-one" onClick = "activate_click()">
+        Add A Thing
+        </button> 
+    </p>
+
+    <a href="about.html">About </a>
```

Comment:
redirect works, but cannot find this link with `document.querySelector("grading-element-two")`